views/dashboardView.handlebars

```diff
@@ -0,0 +1,22 @@
+<form action="/dashboard/{{id}}/edit">
+    <div id="form-container">    
+        <label disabled for="organization-name">Name</label><br>
+        <input disabled type="text" id="organization-name" name="organization-name"><br>
+
+        <label disabled for="organization-description">Description</label><br>
+        <input disabled type="text" id="organization-description" name="organization-description"><br>
+
+        <label disabled for="organization-address">Address</label><br>
+        <input disabled type="text" id="organization-address" name="organization-address"><br>
+
+        <label disabled for="organization-phone">Phone</label><br>
+        <input disabled type="text" id="organization-phone" name="organization-phone"><br>
+
+        <label disabled for="organization-website">Website</label><br>
+        <input disabled type="text" id="organization-website" name="organization-website"><br>
+
+        <label disabled for="organization-email">Email</label><br>
+        <input disabled type="text" id="organization-email" name="organization-email"><br>
+    </div>
+    <button>Edit</button>
```

Comment:
nit: Does the "Edit" button currently redirect when you click it? If you are redirecting using the form it would be good to specify the `method` attribute as `GET`. Could also redirect using a regular anchor tag.

Reply:
changed to use anchor tag